src/main/java/access/Application.java

```diff
@@ -67,14 +108,6 @@ protected SpringApplicationBuilder configure(SpringApplicationBuilder builder) {
 		return builder.sources(Application.class);
 	}
 
-	@Bean
-	public RestTemplate restTemplate() {
-		RestTemplate restTemplate = new RestTemplate();
```

Comment:
I still think that the basic authentication profile will need the HTTP Max Total route configurations/etc. Otherwise we leave ourselves open to load testing problems with reaching the maximum number of sockets.